src/containers/SnippetArea.jsx

```diff
@@ -258,10 +290,19 @@ export class SnippetArea extends React.Component {
             avatarUrl={avatarUrl}
             author={author}
           />
-          <ConfirmLockDialog
+          <ConfirmationDialog
             accept={this.handleToggleReadOnly}
             isOpen={this.state.lockDialogOpen}
             reject={this.handleCloseModal}
+            title="Are you sure you want to lock editing?"
+            message="Note that you will not be able to revert back to edit mode"
+          />
+          <ConfirmationDialog
+            accept={this.handleDelete}
+            isOpen={this.state.deleteDialogOpen}
+            reject={this.handleCloseModal}
+            title="Are you sure you want to delete this snippet?"
+            message="Note that you can not undo this action"
```

Comment:
If we can, it will pay to use one `ConfirmationDialog` component instead of two.